### PR TITLE
clbAuthHttp wrapper

### PR DESCRIPTION
**Pull request by oliamb**
Wrap $http in clbAuthHttp. Any call that authenticate through the HBP Bearer token should use this wrapper instead of calling $http.

### CODE REVIEW

File: src/app/authHttp.service.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+angular.module('clb-app')
+.factory('clbAuthHttp', clbAuthHttp);
+
+/**
+ * Proxy $http to add the HBP bearer token.
+ * Also handle 401 Authentication Required errors.
```
Comment (zanitete) [inline src/app/authHttp.service.js]: how?

Comment (oliamb) [inline src/app/authHttp.service.js]: Not implemented at this time...

**Comment by oliamb** [comment]
I would like to merge this pull request, I will handle the 401 and the doc in another one. Can you review it one last time @zanitete ?

**Comment by zanitete** [comment]
seems ok to me.
A question: where hello.utils is storing info? session/local storage?

I would release it at least as minor release since all the services are now using the new auth.

**Pull request merged**